avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -28,6 +28,20 @@ initialiseParticlesFromFile = False
 particleFile =
 # seed for random generator
 seed = 12345
+# particle initialisation step - set iniStep to True to activate redistribution of particles to reduce SPH force
+iniStep = False
+# artificial viscosity for initialisation parameter
+subgridMixingFactorIni = 10.
+# time step for initialisation
+dtIni = 0.1
+# sphOption for ini
+sphOptionIni = 2
+# stop criterion value for initialisation
+stopCritIni = 0.1
+# max number of iterations
+maxIterations = 100
```

Comment:
could we drop the default for this substantially? Any experiencies/test on this?